compose/ui/ui-text/src/skikoMain/kotlin/androidx/compose/ui/text/SkiaParagraph.skiko.kt

```diff
@@ -57,6 +58,8 @@ internal class SkiaParagraph(
     internal val defaultFont
         get() = layouter.defaultFont
 
+    private val textAlign: TextAlign = layouter.textStyle.textAlign
```

Comment:
No need to put it in a variable, just use it at the one place where it's needed.
It's both safer (no need to understand whether the value can change) and takes less memory.